### DISCUSSION TITLE
$#ndts не добавляет секунды

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#263] [Как это сделать?] [answered]
Создал поле с типом "дата", поставил галочку "дата-время".
Сделал форматирование даты "H:i:s"
В код добавил $#ndts исполняемый при добавлении строки.

В итоге время как ни крути получается время без секунд, по типу 22:43:00 (в конце всегда 00).
Что я не так делаю?
![Screenshot_2](https://user-images.githubusercontent.com/113771444/200937889-ad2d5064-d56f-45f6-a637-297924a2edf7.jpg)

**Comment by erpic-ru** [comment, marked as answer]
Поле дата [не поддерживает секунды](https://ru.docs.totum.online/fields#fields-settings-dateTime). Для секунд следует использовать строковое поле с dateformat по $#ndts

**Reply by ttm-alexeypolunin** [reply]
Угу, у Даты помимо хранения есть датапикер и у него нет секунд.
